pcb/rules/klc_constants.py

```diff
@@ -0,0 +1,23 @@
+
+# silkscreen layer
+KLC_SILK_WIDTH = 0.12
+KLC_SILK_WIDTH_ALLOWED = [0.10, 0.12, 0.15]
```

Comment:
IPC7351C allows for any of those three values. I think we should be permissive here, otherwise we'll never be able to properly conform. Default is still 0.12mm